control/controlclient/auto.go

```diff
@@ -170,73 +253,20 @@ func (c *Auto) Start() {
 //
 // It should be called whenever there's something new to tell the server.
 func (c *Auto) sendNewMapRequest() {
+	at := mono.Now()
```

Comment:
isn't this only like 16ms resolution on Windows?

You'd be better off with an atomic integer sequence number.